test/integration/cli/configuration-cli-test.js

```diff
@@ -54,25 +54,21 @@ function execCommand(custom = {}, cb) {
 describe('CLI class Integration', () => {
   before(() => {
     ['warn', 'error'].forEach((method) => {
-      sinon.stub(loggerStub, method).callsFake((chunk) => { stderr += `\n${method}: ${chunk}`; });
+      sinon
+        .stub(loggerStub, method)
```

Comment:
I think it would be nicer if the loggers we're dependency injected into the reporters. It will make it easier to adapt and simplify the testing removing the need for stubbing.

Reply:
Yes, exactly my wish as well. At this point I'm only separating the loggers, but ideally the reporters would render the output using something completely else than Winston and whatever it is, it would be dependency injected for sure.